metadata/ImageClassification__1__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "ImageClassification",
+    "retrainable": true,
+    "gpu": 1,
+    "processorType": "GPU",
+    "description": "This preview model is a retrainable deep learning model used to classify images. You can train it on your own data and create an ML Skill to perform image classification. This ML Package must be retrained, if deployed without training first, deployment will fail with an error stating that the model is not trained.",
+    "inputDescription": "Full path of the image file which you want to classify. Make sure that the image format is either JPG, JPEG or PNG.",
+    "outputDescription": "JSON with identified label for the image and confidence score (between 0-1).",
+    "changeLog": "Fix dependency bug",
+    "cpu": 1,
+    "inputType": "FILE",
+    "displayName": "ImageClassification",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Image Analysis",
+    "projectDescription": "Curated Models from UiPath to analyze images",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
+    "version": 1,
+    "customVersion": "23.2.0",
+    "contentUri": "https://<placeholder-url>/publicmodels/AIC/ImageClassification/1/img_classifier_package.zip"
```

Comment:
Incorrect

Reply:
@Jemteddy do we delete this file for now?